dependencies.props

```diff
@@ -9,20 +9,20 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>703d40525af4a09f5566cd3d8f334b3736b68fae</CoreFxCurrentRef>
+    <CoreFxCurrentRef>e3392e472dc5b5790ac940459eddb83d42780ce8</CoreFxCurrentRef>
     <CoreClrCurrentRef>fcd99ed86a703a16698fcd0027707739c8ef3501</CoreClrCurrentRef>
-    <StandardCurrentRef>cddd078e3efc27b2fc23c38d8028e3b647cd4fe3</StandardCurrentRef>
-    <WCFCurrentRef>cddd078e3efc27b2fc23c38d8028e3b647cd4fe3</WCFCurrentRef>
+    <StandardCurrentRef>e3392e472dc5b5790ac940459eddb83d42780ce8</StandardCurrentRef>
+    <WCFCurrentRef>e3392e472dc5b5790ac940459eddb83d42780ce8</WCFCurrentRef>
   </PropertyGroup>
   
   <PropertyGroup>
-    <CoreFxVersion>4.4.0-preview2-25316-01</CoreFxVersion>
-    <PlatformPackageVersion>2.0.0-preview2-25316-01</PlatformPackageVersion>
+    <CoreFxVersion>4.5.0-preview2-25319-01</CoreFxVersion>
+    <PlatformPackageVersion>2.1.0-preview2-25319-01</PlatformPackageVersion>
```

Comment:
We are going to need to do this in coreclr as well. In order to get the package validation to work I had to keep it as master until we get a new release/2.0.0 build published